doc/release-notes.md

```diff
@@ -197,6 +197,8 @@ You can't disable HD key generation once you have created a HD wallet.
 
 There is no distinction between internal (change) and external keys.
 
+HD wallets are incompatible with older versions of Bitcoin Core.
```

Comment:
I think you need to open a separate pull against .13 now that the release notes were moved.